### PR TITLE
[BigQuery] Add view details

### PR DESCRIPTION
**Pull request by CynthiaXJia**
Allows a user to double click on a view in the data tree and open a View Details panel with a formatted query.

### CODE REVIEW

File: jupyterlab_bigquery/src/components/details_panel/details_panel.tsx [hunk 1]
```diff
@@ -80,18 +83,63 @@ interface SharedDetails {
...
+        editorElement.style.height = `${height}px`;
+        editor.layout();
+
+        monaco.editor.defineTheme('viewQueryTheme', {
+          base: 'vs',
+          inherit: true,
+          rules: [],
+          colors: { 'editorCursor.foreground': '#FFFFFF' },
+        });
+        monaco.editor.setTheme('viewQueryTheme');
```
Comment (CynthiaXJia) [inline jupyterlab_bigquery/src/components/details_panel/details_panel.tsx:133]: Some hacky stuff here to get a nice view-only editor - there doesn't seem to be a way to disable the cursor so I just made it the same color as the background (white)

File: jupyterlab_bigquery/src/components/details_panel/details_panel.tsx [hunk 2]
```diff
@@ -80,18 +83,63 @@ interface SharedDetails {
...
+    monaco
+      .init()
+      .then(monaco => {
+        const lineHeight = editor.getOption(
+          monaco.editor.EditorOption.lineHeight
+        );
+        const lineCount = editor._modelData.viewModel.getLineCount();
+        const height = lineCount * lineHeight;
+        editorElement.style.height = `${height}px`;
+        editor.layout();
```
Comment (CynthiaXJia) [inline jupyterlab_bigquery/src/components/details_panel/details_panel.tsx:125]: This gets the container to be the correct height, but for some reason there is extra contents beyond the line numbers and actual query text. I will continue investigating, but for now all the relevant query text shows up on the screen ok.

Comment (CynthiaXJia) [reply jupyterlab_bigquery/src/components/details_panel/details_panel.tsx:125]: Aha! Resolved with the editor option `scrollBeyondLastLine: false`

**Comment by cbwilkes** [comment]
There are some conflicts on this branch that I don't know how to resolve. Can you look into it?

**Comment by CynthiaXJia** [comment]
> There are some conflicts on this branch that I don't know how to resolve. Can you look into it?

Should be good to go now!

**Pull request merged**